docs/modules/cockroachdb.md

```diff
@@ -0,0 +1,80 @@
+# CockroachDB
+
+Not available until the next release of testcontainers-go <a href="https://github.com/testcontainers/testcontainers-go"><span class="tc-version">:material-tag: main</span></a>
+
+## Introduction
+
+The Testcontainers module for CockroachDB.
+
+## Adding this module to your project dependencies
+
+Please run the following command to add the CockroachDB module to your Go dependencies:
+
+```
+go get github.com/testcontainers/testcontainers-go/modules/cockroachdb
+```
+
+## Usage example
+
+<!--codeinclude-->
+[Creating a CockroachDB container](../../modules/cockroachdb/examples_test.go) inside_block:runCockroachDBContainer
+<!--/codeinclude-->
+
+## Module reference
+
+The CockroachDB module exposes one entrypoint function to create the CockroachDB container, and this function receives two parameters:
+
+```golang
+func RunContainer(ctx context.Context, opts ...testcontainers.ContainerCustomizer) (*CockroachDBContainer, error)
+```
+
+- `context.Context`, the Go context.
+- `testcontainers.ContainerCustomizer`, a variadic argument for passing options.
+
+When TLS is enabled there's a very small, unlikely chance that the underlying driver can panic when registering the driver as part of waiting for CockroachDB to be ready to accept connections. If this is repeatadly happening please open an issue.
```

Comment:
I'd move this section under the TLS function section.

Also please indent it with:
```suggestion
!!!warning
    When TLS is enabled there's a very small, unlikely chance that the underlying driver can panic when registering the driver as part of waiting for CockroachDB to be ready to accept connections. If this is repeatedly happening please open an issue.
```